### ISSUE TITLE
Messaging: Personless email announcement

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#196]
# Messaging

### Context
We have a launch for personless events coming up. There's an RFC for it here: https://github.com/PostHog/product-internal/pull/583/files so I won't reiterate

This comm needs to go to all users. It needs to go _after_ the TOS update we're doing - https://github.com/PostHog/meta/issues/195 -  Personless is also a blocker to other pricing changes which we'll communicate, so we need to get this out in short order. But deadline is TBD.

I have an art request out with @lottiecoxon - https://github.com/PostHog/posthog.com/issues/8248 - which is in production and will feature here. 

### Requested input
Feedback from @raquelmsmith and @simfish85 

### Previews
First, rough draft. Details subject to change. Non-final art. 

<img width="1270" alt="Screenshot 2024-04-15 at 10 20 15" src="https://github.com/PostHog/meta/assets/84011561/7fe2ed82-7508-4456-87da-615d6eecdcaa">

**Comment by joethreepwood** [comment]
Note: Need to add in the TOS update here too.

**Comment by joethreepwood** [comment]
<img width="1115" alt="Screenshot 2024-04-15 at 11 39 57" src="https://github.com/PostHog/meta/assets/84011561/849ab225-5d38-4f51-a146-7d2a0baed99a">

New version also include TOS news. CC @fraserhopper and @charlescook-ph 

Still not final art or links.

**Comment by joethreepwood** [comment]
<img width="700" alt="Screenshot 2024-04-15 at 13 14 36" src="https://github.com/PostHog/meta/assets/84011561/29fe1dfe-de60-4ccb-845e-245918b85829">

Now featuring the world's fattest pig

**Comment by andyvan-ph** [comment]
- I don't "added a new event type" is correct. Afaik, direction of travel is events are just events, and we're giving people more control over what data is collected. Something like "We're giving you more control of data collection and, good news, it makes PostHog up to 12x cheaper..."

- I think your original final para was better. This one makes it seems like the blog is about our terms.

**Comment by camerondeleone** [comment]
Looks good! Just one small suggestion

> it makes PostHog up to 12x cheaper, depending on your volume
I think you can get rid of "depending on your volume". "up to" already covers that and could read a little overly caveated.

**Comment by joethreepwood** [comment]
<img width="613" alt="Screenshot 2024-04-15 at 16 51 46" src="https://github.com/PostHog/meta/assets/84011561/862e4949-566b-4f8b-9ba1-3071171c8ce2">

**Comment by raquelmsmith** [comment]
nit: I don't think we normally capitalize posthog-js.

**Comment by raquelmsmith** [comment]
I don't think we should call it a "new event type." It should be more around:

> We've split our costs up so you can fine-tune your bill. Now getting advanced person data like person properties and UTMs for never-identified traffic is optional and separate from our event costs.

**Comment by joethreepwood** [comment]
Yep, this terminology was all rough until we had a name. Will respin it
tomorrow now we know it's person profiles (since we at least want to use
that name specifically).

**Comment by corywatilo** [comment]
IMO both this announcement and the [blog post](https://github.com/PostHog/posthog.com/blob/8ae876e44b63457d9be082bbb6952fb3086430d8/contents/blog/cheaper-analytics.md) explain the _how_ but not the _why_. To me we're missing the _story_ that connects the two.

I went ahead and wrote up a blurb to try to explain why we're doing this. Feel free to use here, in the blog post, or ignore entirely.

---

TL;DR: We're making it 12x cheaper to track anonymous users on your website or product by restructuring how we track events.

Most companies track user events in one of two ways (or use both):

- Anonymous tracking (like what Google Analytics collects) - you get basic info like the user's browser, OS, and UTM source
- Identified events (like what PostHog has always offered) - send custom properties (like email, plan, etc) and get a deeper level of insights as a result

Since PostHog began in product analytics, we've only ever offered identified events. This works great for building deep insights, but this per-event pricing doesn't scale with large volumes of traffic (like on a marketing site) where you don't know their email (yet, don't have any custom properties that are worth tracking, and don't get the full value of identifying those visitors.

So today we're introducing an updated event pricing that makes it _cheaper_ to track anonymous visitors. (In fact, if you don't need to identify users, event tracking becomes ~12x cheaper!)

We're also changing how we refer to events:

- All events are now anonymous, and come with basic properties (browser, OS, geo IP, limited UTM properties)
- Enable _Person profiles_ (a new add-on) to get the full power of PostHog events as you've always known them today
  - As an existing customer, we'll enroll you in this automatically, but you can change it at anytime
- Tracking an event + enabling person profiles = the same cost of traditional PostHog events as you've always known them - you won't pay more.

Things you should know:

- You can choose when you want to enable a person profile. A common example of this is when a user signs up (and enters their email), it's a good time to enable their person profile.
- Once a person profile is enabled for a user, you'll be billed for both the (cheaper) event price _and_ for the _person profile_ add-on per-event.

---

Things I tried to hit in the above:

- Why you'd use one vs the other
- "Up to 12x less" is a little disingenuous without directly caveating that it's for anonymous users _only,_ and to use the same product you're used to today, there might not be much difference
- Nomenclature change

**Comment by joethreepwood** [comment]
Thanks all -- appreciate the feedback. Just to clarify: The final version will adhere more closely to James' blog, reference Persons Profiles specifically, explain a bit more of the why, link to finished docs, etc. These are all things we've been blocked on until now and I've been sharing these rough drafts to get feedback from the specific people I've tagged as we continue working on this. Apologies if this wasn't clear before!

**Comment by joethreepwood** [comment]
<img width="562" alt="Screenshot 2024-04-16 at 15 06 17" src="https://github.com/PostHog/meta/assets/84011561/6edfcb64-40b2-438d-980e-bd6b42d36b41">

OK, consider this a first final draft. 

After chatting with a few people I think we can drop the caveat around not being able to use data you don't have, and I've tried to bring the why out a bit more. 

Nomenclature around the config still seems to be changing, but I'll revisit as there'll be at least one more version.

**Comment by robbie-c** [comment]
Where does that link to?

It doesn't flow as well but you could say "you may want to use `identified_only` on your marketing site..." as that's the mode I would expect people to use there.

**Comment by joethreepwood** [comment]
> Where does that link to?

<img width="621" alt="Screenshot 2024-04-16 at 16 02 20" src="https://github.com/PostHog/meta/assets/84011561/7029a306-3fbb-48b8-90f0-ba054deb3c73">

**Comment by joethreepwood** [comment]
> If we wanted to include the min posthog-js version it would be 1.125.0

Do we want to include this? I want to make sure we're giving the correct info in your opinion.

**Comment by joethreepwood** [comment]
<img width="520" alt="Screenshot 2024-04-16 at 17 12 16" src="https://github.com/PostHog/meta/assets/84011561/45b7ddb3-f69d-4bac-a61b-e518b42d7490">

**Comment by joethreepwood** [comment]
<img width="762" alt="Screenshot 2024-06-10 at 15 31 02" src="https://github.com/PostHog/meta/assets/84011561/80ffc018-df2d-40d7-97a6-172ca4090c96">

Now that we're coming up on a louder launch, here's some new suggested copy for the email. CC @raquelmsmith for the approval here.

**Comment by raquelmsmith** [comment]
> If you're like us you probably only need profiles for logged in users, for example. For others you can skip prifiles, track them anonymously in the web analytics beta, and save money in the process. Simple.

Replace with (edits in **bold**):

If you're like us you probably only need profiles for **logged-in** users, for example. For others you can skip **using** profiles **entirely** and track **people** anonymously in the web analytics **beta. For both cases, you'll end up saving** money in the process. Simple.

> ...but if you're curious why we're helping you spend less then CEO James Hawkins

Replace with (edits in **bold**):

...but if you're curious why we're helping you spend **less, CEO** James Hawkins

(the `less then` read very weird in my head, I thought it was a typo at first)

**Comment by joethreepwood** [comment]
<img width="532" alt="Screenshot 2024-06-11 at 11 56 00" src="https://github.com/PostHog/meta/assets/84011561/3334a8d6-7ad6-41be-91e0-cb5cd9053e17">

Great, finalized version. I'll look at the blog post next and we can send when that's locked in.

**Comment by andyvan-ph** [comment]
Missing 'the' before "web analytics beta.

**Comment by joethreepwood** [comment]
I have rethought this from scratch and introduced some much needed sass. 

<img width="733" alt="Screenshot 2024-07-23 at 15 57 51" src="https://github.com/user-attachments/assets/be5afbc0-ff40-42ea-9afd-82f33d3e69f2">

**Comment by joethreepwood** [comment]
Slight tweak

<img width="1190" alt="Screenshot 2024-07-23 at 19 14 19" src="https://github.com/user-attachments/assets/870a7d06-4629-407d-acc8-fba08735239d">

**Comment by corywatilo** [comment]
> Honestly, it's like we're allergic to cash.

Feels forced, but take it or leave it.

> Anonymous events work by skipping the properties we collect on other, identified events

This phrasing feels awkward and doesn't really tell me what "properties" are, or mean to me.

I might be biased, but check out the copy on the [pricing page → explain event types](https://posthog.com/pricing) modal. I put a lot of effort into trying to distill it into something digestible. Maybe we can match some language?

> *Default for unidentified users only, so you save money without much data loss

This even confuses _me_ and I wrote the explanation on this stuff

**Comment by joethreepwood** [comment]
Good notes. Let's try this:

<img width="584" alt="Screenshot 2024-07-24 at 16 22 54" src="https://github.com/user-attachments/assets/39a5f404-c860-456d-94f9-1fbcb303038c">

**Comment by raquelmsmith** [comment]
The first line should say "session replay" - remove the plural S from replays.

I don't want to say we're launching a new event option called anonymous events. We've always had tracking for anonymous users, and I feel like that's inconsistent. But we have is new pricing for anonymous events.

We should also avoid using the term personally identifiable information as that's a legal term, PII, and we should avoid that when we can.

What we do is we skip creating a person in posthog for these anonymous events, and that means there are no properties stored for the person. 

"Personal data" -> person profiles

**Comment by joethreepwood** [comment]
> The first line should say "session replay" - remove the plural S from replays.

✅ 


 
> We should also avoid using the term personally identifiable information as that's a legal term, PII, and we should avoid that when we can.

Switching for 'individually identifiable', which matches the pricing page. Feels a _bit_ clunkier, but legal reason is a good point.

> What we do is we skip creating a person in posthog for these anonymous events, and that means there are no properties stored for the person.


> "Personal data" -> person profiles

I've changed this to 'that data' because I don't think it's useful in the announcement to introduce the concept of person profiles specifically and, if person profiles ultimately are a way of storing this data then referring to the data itself is sufficient. I'd quite strongly like to keep this focused on the use-case and price cut as much as possible, referring to the docs for technicals. 

I basically want to avoid letting this become: "There are anonymous events and identified events. Identified events have person profiles, which are a way of storing person properties, which are individually identifiable data that includes email or other custom properties. Anonymous events skip the person profiles, so don't have person properties, though they do keep some non-customizable data which isn't individually identifiable, such as device and location. Also, anonymous events are cheaper." 🤣  

> I don't want to say we're launching a new event option called anonymous events. We've always had tracking for anonymous users, and I feel like that's inconsistent. But we have is new pricing for anonymous events.

Tried to clear this up. I personally feel it's a _little_ less clear now, but happy to defer

<img width="785" alt="Screenshot 2024-07-26 at 10 20 09" src="https://github.com/user-attachments/assets/2ffcd168-fbae-4829-976a-43c9a88b69f4">

**Comment by simfish85** [comment]
Looks good - not a huge fan of the Oh Crap but maybe it's on brand?

**Comment by raquelmsmith** [comment]
Looks good. As an aside - this isn't an automatic drop for people, because previously-made person profiles will still have events be "identified" even if the person was never "identified." So it'll be more like a slow slide as previously-seen users are replaced with new users. I don't _think_ we need to call that out but I'm calling it out just in case _you_ want to call that out.

**Comment by joethreepwood** [comment]
<img width="585" alt="Screenshot 2024-12-10 at 12 22 46" src="https://github.com/user-attachments/assets/f5502f84-12f1-4f17-b735-91bac8c25e51">

Slight tweak to the language, and also an FYI that we'll send this only to users who match our 'recently engaged' segment (i.e have signed in during the last 4 months) in order to protect our deliverability. I think this classes as a service update, so it'll go to all users regardless of subscription preference. 

Everyone seems happy and the copy changes aren't major, so I'm scheduling this in to send on Wednesday, 11th Dec 11AM London time.
